modrinth error unrelated
```nix
       > error[E0308]: mismatched types
       >   --> crates/turborepo-lib/src/commands/daemon.rs:70:30
       >    |
       > 70 |                 let client = connector.clone().connect().await?;
       >    |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `Result<_, DaemonError>`, found `Result<_, DaemonConnectorError>`
       >    |
       >    = note: expected enum `std::result::Result<_, DaemonError>`
       >               found enum `std::result::Result<_, DaemonConnectorError>`
       > help: use the `?` operator to extract the `std::result::Result<_, DaemonConnectorError>` value, propagating a `Result::Err` value to the caller
       >    |
       > 70 |                 let client = connector.clone().connect().await??;
       >    |                                                                +
       >
       > For more information about this error, try `rustc --explain E0308`.
       > error: could not compile `turborepo-lib` (lib) due to 1 previous error
```

--- a/pkgs/by-name/fi/firezone-gui-client/package.nix
+++ b/pkgs/by-name/fi/firezone-gui-client/package.nix
@@ -43,8 +43,8 @@ let
       inherit pname version;
       pnpm = pnpm_9;
       src = "${src}/rust/gui-client";
-      fetcherVersion = 1;
-      hash = "sha256-ttbTYBuUv0vyiYzrFATF4x/zngsRXjuLPfL3qW2HEe4=";
+      fetcherVersion = 3;
+      hash = "sha256-akanXiWehhQRx6WHN75PjYvFxFiSsJ5dJaguJBTM9J0=";
     };
     pnpmRoot = "rust/gui-client";
 

--- a/pkgs/by-name/fo/folo/package.nix
+++ b/pkgs/by-name/fo/folo/package.nix
@@ -38,8 +38,8 @@ stdenv.mkDerivation rec {
       src
       ;
     pnpm = pnpm_10;
-    fetcherVersion = 1;
-    hash = "sha256-6I10NSmTDd/wmL/HfAgLH+G2MDfuPmrTePNDDy08nRA=";
+    fetcherVersion = 3;
+    hash = "sha256-EP7bpbJUcKmHm7KMlKc0Fz2u0niQ3jC7YN/9pp7vucE=";
   };
 
   env = {

--- a/pkgs/by-name/go/goaway/package.nix
+++ b/pkgs/by-name/go/goaway/package.nix
@@ -28,8 +28,8 @@ let
     pnpmDeps = fetchPnpmDeps {
       inherit (finalAttrs) pname version src;
       sourceRoot = "${finalAttrs.src.name}/client";
-      fetcherVersion = 1;
-      hash = "sha256-cJLJMJNDPr73w5IiB1/zIloIdsUhqI+o/JqoKNwNweI=";
+      fetcherVersion = 3;
+      hash = "sha256-1YZf32lDwX3e9EC0Uixyo+yDczfU4/ZSlG+jFZwf+38=";
     };
 
     pnpmRoot = "client";

